### DISCUSSION TITLE
Rule Name is not useful to find the rule in "Manage correlation rule"

### DISCUSSION THREAD

**Discussion by agauttam** [utmstack/UTMStack#725] [Q&A] [answered]
Hi,

I tried to find the rule using rule name in Manage correlation rules but I could not get the rule.

![image](https://github.com/utmstack/UTMStack/assets/168720506/3928c8ec-234c-4bed-95f8-1cf4dcfe1bc3)

**Comment by c3s4rfred** [comment, marked as answer]
Hi, @agauttam the filter search matches between the value inserted and the file name, not the rule name inside the .yml file.

Best regards
